src/nexus/backend/tensorflow_model.cpp

```diff
@@ -25,15 +25,11 @@ TensorflowModel::TensorflowModel(int gpu_id, const ModelInstanceConfig& config):
   gpu_opt->set_visible_device_list(std::to_string(gpu_id));
   gpu_opt->set_allocator_type("BFC");
   LOG(INFO) << "model memory usage: " << config.memory_usage() << " B";
-//  gpu_opt->set_allow_growth(true);
   if (config.memory_usage() > 0) {
     double memory_usage = config.memory_usage();
     LOG(INFO) << "model memory usage: " << memory_usage << " B";
     gpu_opt->set_per_process_gpu_memory_fraction(
         memory_usage / gpu_device_->TotalMemory());
-    gpu_opt->set_allow_growth(false);
-  } else {
-    gpu_opt->set_allow_growth(true);
```

Comment:
What's the reason for abandoning this branch?

Reply:
Because we don't need `set_allow_growth(true)` now. By default, it is `set_allow_growth(false)`. Besides, the profiler can get a slightly smaller memory usage when set to false.